packages/dev/core/src/Actions/condition.ts

```diff
@@ -135,7 +135,7 @@ export class ValueCondition extends Condition {
         public propertyPath: string,
         /** the value compared by the conditional operator against the current value of the property */
         public value: any,
-        /** the conditional operator, default ValueCondition.IsEqual */
+        /** [number] the conditional operator, default ValueCondition.IsEqual */

```

Comment:
Just curious, who is using the closure compiler? How are these issues being found? Also curious if we want to make sure our code works with closure, are there ESLint plugins that can help with this?

Reply:
answered offline :-)

Reply:
I'm personally curious, too

Reply:
I can answer the how part 😀
We have a CI pipeline running daily that compiles core, loaders, and serializers packages using closure. If it fails, we get notified.